activerecord/test/cases/persistence_test.rb

```diff
@@ -353,12 +353,12 @@ def test_create_many
 
   def test_create_columns_not_equal_attributes
     topic = Topic.instantiate(
-      "attributes" => {
-        "title"          => "Another New Topic",
-        "does_not_exist" => "test"
-      }
+      "title"          => "Another New Topic",
+      "does_not_exist" => "test"
     )
     assert_nothing_raised { topic.save }
```

Comment:
Require `topic = topic.dup # reset @new_record` since `klass.instantiate` always set `"new_record" => false`.

https://github.com/rails/rails/blob/3ddb811acc75292776c39f5a95840b21b32d9fd8/activerecord/lib/active_record/persistence.rb#L71

Reply:
Sorry about that! Fixed.